pkgs/by-name/st/stormchecker/carl.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  boost,
+  gmp,
+  ginac,
+  eigen,
+  replaceVars,
+  gtest,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
https://nixos.org/manual/nixpkgs/stable/#sec-using-stdenv (second codeblock in that chapter)